src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -91,7 +95,10 @@ public void robotInit() {
 
 		if (Subsystems.SubsystemConstants.DRIVEBASE_ENABLED) {
 			if (autoEnabled) {
-				AutoLogic.initShuffleBoard();
+				AutoLogic.initShuffleboard();
+				if (APRILTAGS_ENABLED) {
```

Comment:
Should this also check if drivebase and auto are enabled?

Reply:
Aren't drivebase and auto enabled implicitly checked by the surrounding if-statements?